airbyte-integrations/connectors/source-metabase/metadata.yaml

```diff
@@ -5,26 +5,36 @@ data:
   allowedHosts:
     hosts:
       - "*"
+  connectorBuildOptions:
+    baseImage: docker.io/airbyte/python-connector-base:1.2.0@sha256:c22a9d97464b69d6ef01898edf3f8612dc11614f05a84984451dde195f337db9
   connectorSubtype: api
   connectorType: source
   definitionId: c7cb421b-942e-4468-99ee-e369bcabaec5
-  dockerImageTag: 1.1.0
+  dockerImageTag: 2.0.0
   dockerRepository: airbyte/source-metabase
   documentationUrl: https://docs.airbyte.com/integrations/sources/metabase
   githubIssueLabel: source-metabase
   icon: metabase.svg
   license: MIT
   name: Metabase
-  remoteRegistries:
-    pypi:
-      enabled: true
-      packageName: airbyte-source-metabase
   registries:
     cloud:
       enabled: true
     oss:
       enabled: true
+  releases:
+    breakingChanges:
+      2.0.0:
+        message: "The `dashboards` stream's endpoint has been updated due to the previous endpoint being deprecated by the service. The new version no longer returns the `creator` field for the `Dashboards` stream. After upgrading, please reset the schema for the stream. Additionally, the destination may require updating to handle the updated schema."
```

Comment:
"The `dashboards` stream's endpoint has been updated due to the previous endpoint being deprecated by the service. The new version no longer returns the `creator` field. After upgrading, please reset the schema for the stream. Additionally, the destination may require updating to handle the updated schema."

Reply:
Updated accordingly.